src/Docs/Search/App/Sidebar.purs

```diff
@@ -133,31 +149,62 @@ render { moduleIndex, groupingMode, moduleNames, isIndexHTML } =
   ]
   where
 
-    renderPackageEntry (PackageName packageName /\ modules) =
+    renderPackageEntry (package /\ modules) =
       HH.li [ HP.classes [ wrap "li-package" ] ]
-      [ HH.details_
-        [ HH.summary_ [ HH.text packageName ]
+      [ HH.details
+        (if isCurrentPackage then [ HP.attr (wrap "open") "" ] else [])
+        [ HH.summary_
+          [ HH.text $
+            case package of
+              Package packageName -> unwrap packageName
+              LocalPackage -> unwrap localPackageName
+              Builtin -> "Built-in"
+              UnknownPackage -> "Unknown package"
```

Comment:
This feels a bit inconsistent:
![image](https://user-images.githubusercontent.com/5480361/89732442-48fc7e80-da4f-11ea-9b20-4858f78bd0cb.png)

..as I'd expect everything to be lower case (all package names are lowercased) and "special" packages to be wrapped in `< >`:

```suggestion
              Builtin -> "<builtins>"
              UnknownPackage -> "<unknown>"
```